packages/shared/src/lib/featureManagement.ts

```diff
@@ -1,21 +1,58 @@
 import { IFlags } from 'flagsmith';
 
-export enum Features {
-  SignupButtonCopy = 'signup_button_copy',
-  DevcardLimit = 'feat_limit_dev_card',
-  FeedVersion = 'feed_version',
-  HidePublicationDate = 'hide_publication_date',
-  LoginModalButtonCopyPrefix = 'login_modal_button_copy_prefix',
-  LoginModalDescriptionCopy = 'login_modal_description_copy',
+export class Features {
+  static readonly SignupButtonCopy = new Features(
+    'SignupButtonCopy',
+    'signup_button_copy',
+    'Access all features',
+  );
+
+  static readonly DevcardLimit = new Features(
+    'DevcardLimit',
+    'feat_limit_dev_card',
+  );
+
+  static readonly FeedVersion = new Features(
+    'FeedVersion',
+    'feed_version',
+    '1',
+  );
+
+  static readonly HidePublicationDate = new Features(
+    'HidePublicationDate',
+    'hide_publication_date',
+    '0',
+  );
+
+  static readonly LoginModalButtonCopyPrefix = new Features(
+    'LoginModalButtonCopyPrefix',
+    'login_modal_button_copy_prefix',
+    'Sign in with',
+  );
+
+  static readonly LoginModalDescriptionCopy = new Features(
+    'LoginModalDescriptionCopy',
+    'login_modal_description_copy',
+    'Unlock useful features by signing in. A bunch of cool stuff like content filters and bookmarks are waiting just for you.',
+  );
+
+  private constructor(
+    private readonly key: string,
```

Comment:
Why do we need this property?

Reply:
This actually makes sure we can access them as we did before with `Features.FeatureName`
Just thought it was nicer to use from the developer point of view.

Reply:
Oh I thought that the fact they're defined as `static` already takes care of it, is it not?

Reply:
No it's actually the class called itself with this constructor, so each static is a usage of this class:

```
static readonly LoginModalDescriptionCopy = new Features(
    'LoginModalDescriptionCopy',
    'login_modal_description_copy',
    'Unlock useful features by signing in. A bunch of cool stuff like content filters and bookmarks are waiting just for you.',
  );
```

And the magic is in the `toString` method on the class, making it default return the key.

Reply:
Not sure that we're on the same page. If you define static variable `V` in class `C`, you should be able to access it by using `V.C`. This is why I'm asking why the magic is actually required (it's super cool though!!)

Reply:
omg, haha I was half asleep I guess, removed it now indeed 😂